arkane/ess/gaussian.py

```diff
@@ -376,6 +375,12 @@ def load_energy(self, zpe_scale_factor=1.):
                             elect_energy_source = 'HF'
                     except ValueError:
                         pass
+                elif 'Energy=' in line:
+                    # for xtb
+                    e_elect = float(line.split()[1]) * constants.E_h * constants.Na
+                elif 'Energy=' in line:
+                    # for xtb
+                    e_elect = float(line.split()[1]) * constants.E_h * constants.Na
```

Comment:
This is a duplicate of the above, I think should be removed.

Reply:
Yes, I must have added these in by accident, I think they are not needed.